src/Ryujinx/UI/ViewModels/TitleUpdateViewModel.cs

```diff
@@ -169,7 +169,7 @@ public void SortUpdates()
             }
         }
 
-        private void AddUpdate(string path, bool ignoreNotFound = false)
+        private void AddUpdate(string path, bool ignoreNotFound = false, bool select = false)
```

Comment:
```suggestion
        private void AddUpdate(string path, bool ignoreNotFound = false, bool selected = false)
```

Just to avoid syntax highlighting issues on GitHub.